airflow/contrib/kubernetes/worker_configuration.py

```diff
@@ -367,5 +378,6 @@ def make_pod(self, namespace, worker_uuid, pod_id, dag_id, task_id, execution_da
             affinity=affinity,
             tolerations=tolerations,
             security_context=self._get_security_context(),
-            configmaps=self._get_configmaps()
+            configmaps=self._get_configmaps(),
+            priority_class=environment.get('AIRFLOW_POD_PRIORITY_CLASS', None)
```

Comment:
we can make default as "Preemptible" or remove None entirely. None will not prevent breaking the code and its better if it breaks saying could not get env var . wdyt?

Reply:
I think it would not be right to break the code on the basis of env. It is possible that compute team might allow launching the pods without resources, keeping the default values for such instance on their side.
The default value we will always be setting for the env we are using so we should be good,